### ISSUE TITLE
Not using cuda

### ISSUE THREAD

**Issue by a43992899** [pinokiofactory/yue#7]
Some people report that pinokio version of YuE does not recognize cuda, but comfyui works fine.

https://github.com/multimodal-art-projection/YuE/issues/56

**Comment by BAIS1C** [comment]
yep, 

You are attempting to use Flash Attention 2.0 with a model not initialized on GPU. Make sure to move the model to GPU after initializing it on CPU with `model.to('cuda')`.
Loading checkpoint shards: 100%|████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:00<00:00,  6.27it/s]
C:\Users\b4sic\pinokio\api\yue.git\app\env\Lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
C:\Users\b4sic\pinokio\api\yue.git\app\inference\gradio_server.py:136: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  parameter_dict = torch.load(args.resume_path, map_location='cpu')
************ Memory Management for the GPU Poor (mmgp 3.1.4-15) by DeepBeepMeep ************
You have chosen a profile that requires at least 48 GB of RAM and 24 GB of VRAM. Some VRAM is consumed just to make the model runs faster.
Pinning data of 'transformer' to reserved RAM
The whole model was pinned to reserved RAM: 51 large blocks spread across 11872.51 MB
Hooked to model 'transformer' (LlamaForCausalLM)
Pinning data of 'stage2' to reserved RAM
The whole model was pinned to reserved RAM: 16 large blocks spread across 3743.25 MB
Hooked to model 'stage2' (LlamaForCausalLM)
* Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.

Despite the fact tht nvcc --version shows cuda 11.8 active
